File: DofusApi/Models/Equipment.cs
using System.Collections.Generic;

namespace DofusApi.Models
{
    public class Equipment
    {
        public int ID { get; set; }
        public int AnkamaID { get; set; }
        public string Name { get; set; }
        public int Level { get; set; }
        public string Type { get; set; }
        public string ImgUrl { get; set; }
        public string Url { get; set; }
        public string Description { get; set; }
        public ICollection<EquipmentStatistic> EquipmentStatistics { get; set; }
        public ICollection<EquipmentRecipe> EquipmentRecipes { get; set; }
        
        
    }
}

File: DofusSpa/src/app/_models/equipment.ts
import { EquipmentRecipe } from "./equipment-recipe";
import { EquipmentStatistic } from "./equipment-statistic";

export interface Equipment{
    id : number,
    ankamaID : number,
    name : string,
    level : number , 
    type : string,
    imgUrl : string,
    url : string,
    description : string,
    statistics : EquipmentStatistic[],
    recipes : EquipmentRecipe[]
}